sw/device/silicon_creator/rom/doc/shutdown.md

```diff
@@ -0,0 +1,1584 @@
+# Mask ROM Shutdown Module
+
+## Objective
+
+The Shutdown module is responsible for securely shutting down the OpenTitan chip in the event of an unrecoverable fault during the secure boot process.
+A secure shutdown should result in a system end-state where a chip reset is the only viable option.
+
+## Dependencies
+
+Hardware:
+*   [Alert Handler](https://docs.opentitan.org/hw/ip/alert_handler/doc/)
+*   OTP Controller & Shutdown OTP policy words defined.
+*   Lifecycle controller
+
+Software:
+*   Alert Handler Driver
+*   OTP Driver
+*   PMP
+*   Lifecycle Driver
+
+## Implementation Details
+
+The shutdown module will make use of the alert handler to detect shutdown-worthy events within the hardware.
+Hardware and software shutdown events will use the alert manager to cause an escalation resulting in chip shutdown.
+
+### Alert Class Specification
+
+The Mask ROM will categorize shutdown events into the categories shown below.
+OTP words will be used to configure how the ROM will configure Alert Handler’s response to each class of alert.
+
+<table>
```

Comment:
If possible, it would be nice to have these as markdown tables so they're readable from the text file and rendered with the same style as other tables.

The multi-row headers used in one of these tables isn't supported, however.

I know it's tedious to convert these HTML tables, so here's a patch if you choose to use it: https://gist.github.com/jwnrt/2f624351073631febffa8687212fe88b

Reply:
Fixed.